### DISCUSSION TITLE
How to solve this problem？

### DISCUSSION THREAD

**Discussion by Seatosoak** [PaddlePaddle/PaddleOCR#6709] [Q&A] [answered]
**`ValueError: You are going to using gloo on windows, but currently is not supported`**
![1](https://user-images.githubusercontent.com/94884888/175846569-6a0fcd62-4c8d-4977-acca-8572ee4c240a.jpg)

**Comment by andyjiang1116** [comment]
which paddle version do you use?

**Reply by Seatosoak** [reply]
I both used paddlepaddle 2.3.0 and paddlepaddle-gpu 2.3.0，but all give the same mistake.

**Comment by andyjiang1116** [comment, marked as answer]
Could you show me the command you used?

**Reply by Seatosoak** [reply]
您好，现在没有报这个错了，我复现这个代码，
python 3.8.5
paddelpadle-gpu 2.3.0
cuda 11.1
touch 1.10.0+cu113
运行命令`python -m paddle.distributed.launch --gpus 0 tools/train.py -c configs/rec/rec_icdar15_train.yml`
然后报了`ABORT!!! Out of all 1 trainers,the trainer process with rank=[0O] wasaborted. Please check its log.`的错。(如下图)
![YD1}0K10N7QL$4BN$YJ6Z`W](https://user-images.githubusercontent.com/94884888/176082271-5fe1b384-b919-4adf-8d97-aa22110ef77b.png)
网上也没有解决的方法，希望您能拨冗指点一下，万分感谢。

**Reply by andyjiang1116** [reply]
When you use single gpu to train, just use command `python tools/train.py -c configs/rec/rec_icdar15_train.yml`, besides,  distributed training is not supported on  Windows

**Reply by Seatosoak** [reply]
救命，你是我的神！问题解决了，万分感谢！

**Reply by Hwd418** [reply]
如果是使用cpu该怎么办？
